character-controller.js

```diff
@@ -690,34 +733,34 @@ class StatePlayer extends PlayerBase {
             if (!cancelFn.isLive()) return;
             _setNextAvatarApp(nextAvatarApp);
           } else {
-            console.warn('switching avatar to instanceId that does not exist in any app manager', instanceId);
+            _setNextAvatarApp(null)

```

Comment:
I think it's a legit state error if you try to switch to a nonexistent avatar and we shouldn't ignore that.

If the user wanted to set a null avatar they could do so explicitly.